src/design-system/Modal.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Colors from "./Colors";
+import ModalDialog from "./ModalDialog";
+
+const ModalContainer = styled.div``;
+
+const ModalTrigger = styled.h1`
+  color: ${Colors.green};
+  cursor: pointer;
+  font-family: 'Libre Baskerville', serif;
+  font-size: 32px;
+  line-height: 32px
+  letter-spacing: -0.03em
+`;
+
+interface Props {
+  modalTitle?: string;
```

Comment:
you could export this prop from the child component that cares about it so that you don't have to define it twice and keep the two in sync as they change

Reply:
I'm not exactly sure what that would look like, when you have time maybe we could have a quick chat about it?